src/components/buttonMenu/buttonMenu.svelte

```diff
@@ -31,6 +31,11 @@
     if (isOpen === undefined) isOpenInternal = toggleTo
     onChange?.({ isOpen: toggleTo })
   }
+
+  const handleClose = (e: CloseEventDetail) => {
+    if (isOpen === undefined) isOpenInternal = false
+    onClose?.(e)
+  }
```

Comment:
the state wasn't being updated onClose and therefore the menu took multiple clicks to open again?

Reply:
Exactly, yeah :smile: